packages/editor/src/store/effects.js

```diff
@@ -144,7 +145,10 @@ export default {
 	REQUEST_POST_UPDATE: ( action, store ) => {
 		requestPostUpdate( action, store );
 	},
-	REQUEST_POST_UPDATE_SUCCESS: requestPostUpdateSuccess,
```

Comment:
Is the old effect `requestPostUpdateSuccess` not necessary anymore?

`updateViewAsPermalink` seems undefined right?